builds/running-entitled-builds.adoc

```diff
@@ -10,16 +10,10 @@ include::modules/builds-create-imagestreamtag.adoc[leveloffset=+1]
 
 include::modules/builds-source-secrets-entitlements.adoc[leveloffset=+1]
 
-There are two paths to pulling in the base RHEL image:
-
-* Add the pull secret to registry.redhat.io to your project.
-* Create an imagestream in the `openshift` namespace for the RHEL-based
-image. This makes the imagestream available across the cluster.
+To pull in the base RHEL image, create an imagestream in the `openshift` namespace for the RHEL-based image. This makes the imagestream available across the cluster.
```

Comment:
I would tweak this slightly.

"The recommended approach is to create an imagestream that references the base RHEL image in the `openshift` namespace.  The benefits of this are 

1) Imagestreams in the openshift namespaces make their associated images available to developers in any namespace in the cluster
2) Imagestreams benefit automatically from having access to the install pull secret, which includes registry.redhat.io

Otherwise, if you just pull the image, you have to install your own pull secret with registry.redhat.io credentials into the namespace your are working in, and developers in each namespace have to so the same.

Of course, if you want to restrict which namespaces have access, you could also just create imagestreams in those namespaces, and still benefit from being able to acccess registry.redhat.io credentials from the install pull secret, but then restrict who has access to the credentials."

Reply:
PTAL at the updated text.

Reply:
> I would tweak this slightly.
Please take a look at the updated text I'm proposing. If that's too far off, I'll use the text you've suggested above (or something very close to it).